From: Soren Andersen <san@skov.com>
Date: Tue, 21 Sep 2021 08:30:22 +0200
Subject: [PATCH] [CORE-9371] make sure boards with usb to Ethernet are NFS
 booted with an extra parameter ip=:::::eth1:dhcp             Otherwise it
 takes too long to boot, if we should wait for an eth0 timeout

Signed-off-by: Soren Andersen <san@skov.com>
---
 arch/arm/boards/skov-imx6/defaultenv-skov-imx6/network/eth1-discover | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boards/skov-imx6/defaultenv-skov-imx6/network/eth1-discover b/arch/arm/boards/skov-imx6/defaultenv-skov-imx6/network/eth1-discover
index e11a3f900671..bbd7c47b0ac9 100644
--- a/arch/arm/boards/skov-imx6/defaultenv-skov-imx6/network/eth1-discover
+++ b/arch/arm/boards/skov-imx6/defaultenv-skov-imx6/network/eth1-discover
@@ -5,4 +5,5 @@ if [ "$eth0.mode" != "disabled" ]; then
 	exit 0;
 fi
 
+global.linux.bootargs.base="${global.linux.bootargs.base} ip=:::::eth1:dhcp"
 usb
